service/src/main/resources/schema/development/schema.json

```diff
@@ -180,6 +191,19 @@
           "type": "integer"
         }
       }
+    },
+    "samples": {
+      "type": "object",
+      "properties": {
+        "disease": {
+          "type": "array",
```

Comment:
For this, we should use https://www.ebi.ac.uk/ols/ontologies/mondo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FMONDO_0000001&lang=en&viewMode=PreferredRoots&siblings=false